src/olympia/devhub/forms.py

```diff
@@ -129,6 +129,12 @@ def create_option(self, name, value, label, selected, index,
 
 
 class LicenseForm(AMOModelForm):
+    # Hack to restore behavior from pre Django 1.10 times.
+    # Django 1.10 enabled `required` rendering for required widgets. That
+    # wasn't the case before, this should be fixed properly but simplifies
+    # the actual Django 1.11 deployment for now.
+    use_required_attribute = False
```

Comment:
If we're going to eventually fix this, can we log the "fix properly" issue and link to it here?

Reply:
It's https://github.com/mozilla/addons-server/issues/8912 but I've just copy/pasted the existing comment... I'll update it